kratos/sources/kernel.cpp

```diff
@@ -18,11 +18,11 @@
 namespace Kratos {
 Kernel::Kernel() : mpKratosCoreApplication(Kratos::make_shared<KratosApplication>(
                 std::string("KratosMultiphysics"))) {
-    std::cout << " |  /           |             " << std::endl;
-    std::cout << " ' /   __| _` | __|  _ \\   __|" << std::endl;
-    std::cout << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
-    std::cout << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
-    std::cout << "           Multi-Physics " << KRATOS_VERSION << std::endl;
+    KRATOS_INFO("") << " |  /           |             " << std::endl;
+    KRATOS_INFO("") << " ' /   __| _` | __|  _ \\   __|" << std::endl;
+    KRATOS_INFO("") << " . \\  |   (   | |   (   |\\__ \\ " << std::endl;
+    KRATOS_INFO("") << "_|\\_\\_|  \\__,_|\\__|\\___/ ____/" << std::endl;
+    KRATOS_INFO("") << "           Multi-Physics " << KRATOS_VERSION << std::endl;
```

Comment:
I know that this is not a performance bottleneck point but to make it in a good way you should avoid using KRATOS INFO macro in each line and calling it once in order to remove unnecessary performance overhead and preventing also any other message interlace your message lines.